ui/src/app/utils/access-utils.spec.tsx

```diff
@@ -131,7 +256,7 @@ describe('useIsUserDisabled', () => {
   };
 
   const simulateSignIn = async(wrapper: ReactWrapper, isSignedIn: boolean) => {
-    authStore.set({authLoaded: true, isSignedIn});
+    authStore.set({authLoaded: true, isSignedIn, authError: ''});
```

Comment:
makes IntelliJ happy

Reply:
I think a better fix for this would be to change the interface definition for `Authstore`: `authError: string` --> `authError?: string`